src/main/java/com/futurecollars/lesson7/zad2/Movie.java

```diff
@@ -0,0 +1,15 @@
+package com.futurecollars.lesson7.zad2;
+
+public class Movie {
+    private final String title;
+    private final String nameAndSurname;
```

Comment:
zbyt ogólne, z tej nazwy rozumiem, że jest to imię i nazwisko filmu? brakuje słowa reżyser, zamiast nameAndSurname lepiej dać fullName